File: datasets/IR-Plag/convertai-original-java-rest-python/case-02/plagiarized/L2/05/inputoutput.py
import math

# masukan silinder
radius = float(input("Enter the radius and length of a cylinder: "))
length = float(input())

# perhitungan
area = radius ** 2 * math.pi
volume = area * length

# cetak hasil
print(f"The area is {area:.5f}")
print(f"The volume of the cylinder is {volume:.5f}")

File: datasets/IR-Plag/convertai-original-java-rest-kotlin/case-02/plagiarized/L6/05/inputoutput.kt
package IR

import java.util.Scanner

object inputoutput {
    private var rad: Double = 0.0
    private var panjang: Double = 0.0
    private val masukan = Scanner(System.`in`)

    fun input() {
        rad = masukan.nextDouble()
        panjang = masukan.nextDouble()
    }

    fun hitArea(): Double {
        return Math.pow(rad, 2.0) * 3.14159
    }

    @JvmStatic
    fun main(args: Array<String>) {
        // masukan silinder
        println("Enter the radius and length of a cylinder: ")
        input()

        val vol: Double
        val a: Double
        //perhitungan
        a = hitArea()
        vol = a * panjang

        // cetak hasil
        println("The area is $a")
        println("The volume of the cylinder is $vol")
    }
}

